.github/workflows/build.yml

```
@@ -128,10 +128,29 @@ jobs:
           GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
           GPG_PASSWORD: ${{ secrets.GPG_PASSWORD }}
 
+  build-graal:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          version: 'latest'
+          java-version: '11'
+          components: 'native-image'
```

I think this should fix the rate limit errors:

```suggestion
          components: 'native-image'
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

---

Thanks!